server/src/controllers/job_posts/getOneJobPost.js

```diff
@@ -0,0 +1,42 @@
+const { query } = require("express");
```

Comment:
Same thing as above - I think query is a part of the 'req' object and doesn't need to be explicitly imported. However, it's fine to leave it or remove it!